cmd/lakefs/cmd/run.go

```diff
@@ -425,21 +425,13 @@ func checkRepos(ctx context.Context, logger logging.Logger, authMetadataManager
 }
 
 func scheduleCleanupJobs(ctx context.Context, s *gocron.Scheduler, c *catalog.Catalog) error {
-	const (
-		deleteExpiredLinkAddressesInterval = 3 * ref.LinkAddressTime
-		deleteExpiredTaskInterval          = 24 * time.Hour
-	)
+	const deleteExpiredTaskInterval = 24 * time.Hour
 
 	jobData := []struct {
 		name     string
 		interval time.Duration
 		fn       func(context.Context)
 	}{
-		{
-			name:     "delete expired link addresses",
```

Comment:
Minor: Some tokens will be kept forever in the kv..

Reply:
I think we shouldn't invest on the cleanup as this will probably won't make a dent in the DB